core/templates/dev/head/pages/exploration_player/ExplorationPlayerStateService.ts

```diff
@@ -147,11 +181,22 @@ oppia.factory('ExplorationPlayerStateService', [
           initExplorationPlayer(callback);
         }
       },
+      initializeQuestionPlayer: function(config, callback) {
+        PlayerTranscriptService.init();
+        initQuestionPlayer(config, callback);
+      },
       getCurrentEngineService: function() {
         return currentEngineService;
       },
       isInPretestMode: function() {
-        return inPretestMode;
+        return explorationMode === EXPLORATION_MODE.PRETEST;
+      },
+      isInQuestionMode: function() {
```

Comment:
I'm not sure about the naming here, what's the diff in  `isInQuestionMode` and `isInQuestionPlayerMode`?

Reply:
isInQuestionMode includes tests that are in the context of the exploration (i.e. pretests/ and maybe review tests)
isInQuestionPlayerMode includes tests outside of the context of an exploration (like practice sessions)